docs/getting-started.md

```diff
@@ -478,11 +468,13 @@ The above command will automatically run your app on the iOS Simulator by defaul
 
 <h2>Running your React Native application</h2>
 
-Run `react-native run-android` inside your React Native project folder:
+> If you use Yarn package manager, your can use `yarn` instead of `npx` when running React Native commands inside an existing project.
```

Comment:
```suggestion
> If you use the Yarn package manager, you can use `yarn` instead of `npx` when running React Native commands inside an existing project.
```